ocean_grid_generator.py

```diff
@@ -1448,6 +1449,71 @@ def main(
     if y3.shape[0] % 2 == 0:
         raise Exception("Ooops: The number of j's in the supergrid is not even. Use option --south_cutoff_row to one more or on less row from south.")
 
+    # south-pad, so NYGLOBAL hits a highly composite number
+    def next_composite_number(n, *, min_n=120, max_n=4000, divisor=12):
```

Comment:
This functionality is neat, but my opinion is that it makes the behaviour unclear and somewhat uncontrollable because the arguments to `next_composite_number` cannot be set at the command line. I think it would be cleaner just to let the user decide offline what a good target NY is and use that without modification.